src/routes/ticketcontrol/+page.svelte

```diff
@@ -59,17 +75,80 @@
 
         const url = new URL('https://657cb0cd853beeefdb99d741.mockapi.io/tickets');
         const decodedTextToString = decodedText.toString();
-        //     url.searchParams.append('qrcode', decodedTextToString);
-        console.log("URL", url);
-        console.log("DecodedText", decodedText);
-        console.log("DecodedToString", decodedTextToString);
+        // url.searchParams.append('qrcode', decodedTextToString);
+        //console.log("URL", url);
+        //console.log("DecodedText", decodedText);
+        //console.log("DecodedToString", decodedTextToString);
+        let ticket;
+
+        await getTicket().then((tickets) =>{
+            let ticketArray = tickets
+            for(let i=0; i<ticketArray.length; i++){
+                if(ticketArray[i].qrcode === decodedTextToString){
+                    ticket = ticketArray[i];
+                }
+            }
+
+                    const movieTitle = `<div>MovieTitle:  ${ticket.movieTitle}</div>`;
```

Comment:
here u can use sveltekit real good, and dont need to add them do the container1, with a $: var or with a if condition or smth else